src/core-class-example.js

```diff
@@ -0,0 +1,27 @@
+class Module {
+  constructor(command) {
+    if (!command || typeof command !== 'string') {
+      throw Error('Command must be a string.');
+    }
+    this.command = command;
+    this.enabled = true;
+  }
+
+  onReceive() {
```

Comment:
We have arguments and we also need a `sender` object.

Reply:
We have to make `onReceive()` method be required as we can't make modules without it.
Can we use `abstract` classes with one `abstract onReceive()` method and two non-abstract `onEnable()`/`onDisable()` methods in JS?

Reply:
No, there is no abstract classes. Any class inherited from `Module` will have the default implementation of `onReceive`, `onDisable` and `onEnable` hooks. User can override them if it is necessary.

Reply:
Added `sender` parameter to all of the hooks.